benchmarking/gpt-oss-rhoai-compare/README.md

```
@@ -0,0 +1,274 @@
+# RHOAI 3.4
+mkdir -p rhoai_3.4
+git clone --depth 1 --branch 'v0.7.1+rhaiv.1' https://github.com/opendatahub-io/ogx.git rhoai_3.4/ogx
+
+# BFCL Gorilla
+git clone https://github.com/ShishirPatil/gorilla.git gorilla
+cd gorilla && git checkout 6ea57973c7a6097fd7c5915698c54c17c5b1b6c8 && cd ..
+```
+
+Local folder structure:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)